### DISCUSSION TITLE
Installing on Chrome OS

### DISCUSSION THREAD

**Discussion by ghost** [opensteno/plover#1299] [Q&A and Support]
Hello! I'm a new steno user, slowly learning the ropes. My most common computer, which I use to take minutes of meetings, is a Chrome OS laptop, HP Chromebook 14-CA. I tried installing Plover through Linux through a few means, including compiling from the tar.gz file (but there seems to be a problem with python2), the .deb file, which installs properly but does not launch, and the .appimage file, which gives a qt error saying no qt framework could be initialized. 

It is not strictly necessary to be able to install it, but it would be nice. Does anyone have any experience with this, and could you help me? I'll include my machine details below:

`Chrome OS 89.0.4389.130`
`Intel Celeron CPU N3350 @ 1.10 GHz`
`x86-64`
`4GB Ram`

Thank you in advance!

**Comment by user202729** [comment]
**edit**: See also [Linux Chrome OS support · Issue #1073 · openstenoproject/plover](https://github.com/openstenoproject/plover/issues/1073)

--------

I don't really use Chrome OS, but if it can run Android programs, Dotterel should work.

Also I can find one post on Plover Google Group about installing on a Chromebook: https://groups.google.com/d/topic/ploversteno/lTD_82EH6ms 

If nothing works, you can still buy a Raspberry Pi / similar small computers and run a "Plover-in-the-middle" (firmware) approach.

**Reply by ghost** [reply]
Thanks so much for your help!
